nym-vpnd/src/command_interface/listener.rs

```diff
@@ -39,6 +49,8 @@ impl CommandInterface {
     ) -> Self {
         Self {
             vpn_command_tx,
+            status_tx: tokio::sync::broadcast::channel(10).0,
+            _connection_state_tx: tokio::sync::broadcast::channel(10).0,
```

Comment:
that looks rather iffy to put channels there for the sake of putting them (since we're dropping half of them immediately); perhaps the `CommandInterface` should hold an `Option`?